src/binding.cc

```diff
@@ -1013,6 +1015,31 @@ class ExternalStaticOneByteStringResource
   const int _length;
 };
 
+class ExternalConstOneByteStringResource
```

Comment:
Maybe worth adding a note that this class is never actually used -- it only serves as a c++ reference to the one created on the Rust side.